### DISCUSSION TITLE
Source of Output/Input tabs

### DISCUSSION THREAD

**Discussion by korpe1** [wwmm/easyeffects#3275] [Q&A]
Where do the output/input tabs gets details for the list of ongoing playbacks? Is it MPRIS (Media Player Remote Interfacing Specification) or the Pipewire server? If the Pipewire server, what parameters a node need to have to get included the list?

**Comment by violetmage** [comment]
Easyeffects gets all information directly from the pipewire server.

Edit: see @wwmm's answer

**Comment by wwmm** [comment]
As long as the playback stream has the tag https://github.com/wwmm/easyeffects/blob/900c5fc23c100f124aa1d358f5165b4bf64b27ff/include/tags_pipewire.hpp#L42 set it should be visible in EasyEffects. PipeWire usually sets this for all playback streams as far as I know.
